runtime/runtime.go

```diff
@@ -16,8 +18,14 @@ type Runtime struct {
 	ReadonlyDB        *sql.DB
 	RP                *redis.Pool
 	ES                *elasticsearch.TypedClient
+	FCM               FCMClient
```

Comment:
I think we need the authclient for the verification part as well as currently we initialize that mutliple times in `VerifyFCM`